metrics/prometheus.go

```diff
@@ -1907,6 +1907,18 @@ func (c *PrometheusCollector) collectContainersInfo(ch chan<- prometheus.Metric)
 				}
 			}
 		}
+		if c.includedMetrics.Has(container.PerfMetrics) {
+			perfErrors = append(perfErrors, cont.Stats[0].Perf.PerfErrors...)
```

Comment:
Is is checked somehow that cont.Stats contains at least one element?

Reply:
It is checked in line [1878](https://github.com/kwisniewski98/cadvisor/pull/7/files#diff-88eab3cc8cef9ad727beea9923056cbbR1878)